src/test/java/org/fundacionjala/coding/Raul/test/ChildrenMovieTest.java

```diff
@@ -0,0 +1,28 @@
+package org.fundacionjala.coding.Raul.test;
+
+import org.fundacionjala.coding.raul.movies.ChildrenMovie;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class ChildrenMovieTest {
+    private static final int PRICE = 13;
+    private static final int PRICECODE = 13;
+    private static final double INCREMENT = 1.5;
+    private final ChildrenMovie childrenMovie = new ChildrenMovie("Lion", PRICE);
```

Comment:
Don't instantiate in the attribute of class
Please use a @Before method

Reply:
Done